core/storage/user/gae_models_test.py

```diff
@@ -236,7 +253,8 @@ def test_export_data_for_nontrivial_case_returns_data_correctly(self):
         self.assertEqual(expected_user_data, user_data)
 
     def test_get_new_id_under_normal_behaviour_returns_unique_ids(self):
-        ids = set([])
+        # type: () -> None
+        ids = set([]) # type: Set[Text]
```

Comment:
Creating a set by casting empty list doesn't make much sense here. We could just use `ids = set()` too, right? Though I think you would still need the type hint.

Reply:
Yes set() is same is set([]). Changed it!